serving/docker/partition/trt_llm_partition.py

```diff
@@ -1,39 +1,190 @@
+            "If want to get the accepted tokens' logits from target model, please just enable gather_generation_logits"
+        )
+
+    logger.set_level(args.log_level)
+    tik = time.time()
+
+    if hasattr(args, 'paged_kv_cache'):
+        logger.warning(
+            'Option --paged_kv_cache is deprecated, use --kv_cache_type=paged/disabled instead.'
+        )
+
+    plugin_config = PluginConfig.from_arguments(args)
+    plugin_config.validate()
+    if args.fast_build:
+        plugin_config.manage_weights = True
+
+    speculative_decoding_mode = SpeculativeDecodingMode.from_arguments(args)
+
+    if args.build_config is None:
+        if args.multiple_profiles == "enable" and args.opt_num_tokens is not None:
+            raise RuntimeError(
+                "multiple_profiles is enabled, while opt_num_tokens is set. "
+                "They are not supposed to be working in the same time for now."
+            )
+        if args.cluster_key is not None:
+            cluster_config = dict(cluster_key=args.cluster_key)
+        else:
+            cluster_config = infer_cluster_config()
+
+        # This should only be used for debugging.
+        # The env var BUILDER_FORCE_NUM_PROFILES should override the number of
+        # optimization profiles during TRT build.
+        # BUILDER_FORCE_NUM_PROFILES must be less than or equal to the number of
+        # optimization profiles set by model's prepare_inputs().
+        force_num_profiles_from_env = os.environ.get(
+            "BUILDER_FORCE_NUM_PROFILES", None)
+        if force_num_profiles_from_env is not None:
+            logger.warning(
+                f"Overriding # of builder profiles <= {force_num_profiles_from_env}."
+            )
 
+        build_config = BuildConfig.from_dict(
+            {
+                'max_input_len': args.max_input_len,
+                'max_seq_len': args.max_seq_len,
+                'max_batch_size': args.max_batch_size,
+                'max_beam_width': args.max_beam_width,
+                'max_num_tokens': args.max_num_tokens,
+                'opt_num_tokens': args.opt_num_tokens,
+                'max_prompt_embedding_table_size':
+                args.max_prompt_embedding_table_size,
+                'gather_context_logits': args.gather_context_logits,
+                'gather_generation_logits': args.gather_generation_logits,
+                'strongly_typed': True,
+                'force_num_profiles': force_num_profiles_from_env,
+                'weight_sparsity': args.weight_sparsity,
+                'profiling_verbosity': args.profiling_verbosity,
+                'enable_debug_output': args.enable_debug_output,
+                'max_draft_len': args.max_draft_len,
+                'speculative_decoding_mode': speculative_decoding_mode,
+                'input_timing_cache': args.input_timing_cache,
+                'output_timing_cache': args.output_timing_cache,
```

Comment:
Let's provide a default value for `output_timing_cache` or just hardcode it as `/tmp/cache.engine`

Reply:
went with hardcode of `/tmp/model.cache`